### DISCUSSION TITLE
obsidian组织笔记的N个 xx first

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#37] [备发布站的评论系统]
# obsidian组织笔记的N个 xx first

  [[link first]]  [[local first]]  [[folder first]]  [[dailynote first]]  [[material first]]  [[topic first]]  [[architect first]]  [[action first]]

https://notes.oldwinter.top/obsidian%E7%BB%84%E7%BB%87%E7%AC%94%E8%AE%B0%E7%9A%84n%E4%B8%AA-xx-first

**Comment by Neal-799** [comment]
记得填坑！！【狗头
